### PR TITLE
Wangyif/p2p measurement v0.1

### PR DESCRIPTION
**Pull request by wangyf2001**
Optimized the code format of bwmurclient.py and added descriptions of program measurement data and output results in the README.md file.

### CODE REVIEW

File: bwmur/bwmurclient.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+                data["bandwidth"] = float(result.split()[5])
+                data["time"] = get_datetime()
+        ltcstatus = os.popen(" ".join(latency_mur_cmd)).read()
+        doc = open('clientbwlog.txt', 'a+')
+        doc.write(get_datetime())
+        doc.write("\n")
+        doc.write(" ".join(latency_mur_cmd))
+        doc.write("\n")
+        doc.write(ltcstatus)
+        for i in range(1,11):
```
Comment (zhixiongniu) [inline bwmur/bwmurclient.py]: Should not a fixed number here. Should define a variable.

File: bwmur/bwmursever.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+import sys
+from scp import SCPClient
+import numpy as np
+import copy
+import time
+
+socket_timeout_sec = 240
+machines_file = "machines.json"
+sever_port = "8000"
+sever_ip = "20.81.187.38"
```
Comment (zhixiongniu) [inline bwmur/bwmursever.py]: Should not a fixed IP here.

File: bwmur/bwmurclient.py [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+import json
+import os
+import sys
+import re
+import copy
+import time
+
+socket_timeout_sec = 240
+machines_file = "machines.json"
+sever_port = "8000"
```
Comment (zhixiongniu) [inline bwmur/bwmurclient.py]: Typo

**Review by zhixiongniu — CHANGES_REQUESTED** [review]

File: bwmur/bwmurclient.py (hunk 1 above)
Comment (wangyf2001) [reply bwmur/bwmurclient.py]: Defined the promotion variable and assigned it a value of 11.

File: bwmur/bwmursever.py (hunk 2 above)
Comment (wangyf2001) [reply bwmur/bwmursever.py]: Change it to 127.0.0.1.

File: bwmur/bwmurclient.py (hunk 3 above)
Comment (wangyf2001) [reply bwmur/bwmurclient.py]: Change all sever -> server.

File: bwmur/bwmursever.py (hunk 2 above)
Comment (zhixiongniu) [reply bwmur/bwmursever.py]: to 0.0.0.0?

**Review by zhixiongniu — APPROVED** [review]

File: bwmur/bwmursever.py (hunk 2 above)
Comment (wangyf2001) [reply bwmur/bwmursever.py]: OK, I changed it to 0.0.0.0.

**Pull request merged**